packages/core/src/sequelize.js

```
@@ -244,6 +277,107 @@ export class Sequelize extends SequelizeTypeScript {
+        if (!queryOptions.attributes) {
+          queryOptions.attributes = Array.from(model.modelDefinition.attributes.keys());
+          queryOptions.originalAttributes = model._injectDependentVirtualAttributes(
+            queryOptions.attributes,
+          );
+        }
+
+        const attrMap = model.modelDefinition.attributes;
+        const columns = queryOptions.attributes.map(attr => {
+          const attrName = Array.isArray(attr) ? attr[0] : attr;
+          const def = attrMap.get(attrName);
+          return { name: attrName, dataType: def?.type ?? null };
```

ESLint

[error] Expected blank line before this statement.

(padding-line-between-statements)